backup/pom.xml

```diff
@@ -0,0 +1,136 @@
+        <rabbitmq.version>3.5.4</rabbitmq.version>
+        <thrift.version>0.9.3</thrift.version>
+        <azure.svc-mgmt.version>0.9.3</azure.svc-mgmt.version>
+        <azure.ad.auth.version>1.1.2</azure.ad.auth.version>
+        <azure.rsrc-mgmt.version>1.0.3-SNAPSHOT</azure.rsrc-mgmt.version>
+        <azure.storage.version>5.1.1</azure.storage.version>
+        <shiro.version>1.2.2</shiro.version>
+        <cassandra.driver.version>3.2.0</cassandra.driver.version>
+        <zendesk.client.version>0.5.4</zendesk.client.version>
+        <jira.version>0.5</jira.version>
+
+    </properties>
+
+    <artifactId>backup</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.9</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+            <version>4.2.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.inject.extensions</groupId>
+            <artifactId>guice-assistedinject</artifactId>
+            <version>4.2.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>24.1-jre</version>
+        </dependency>
+
+        <!--TODO: change to BOM style to only include s3 deps -->
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk</artifactId>
+            <version>1.11.106</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+
+        <!-- Azure Storage SDK -->
+        <!-- https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage -->
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-storage</artifactId>
+            <version>7.0.0</version>
+        </dependency>
+
+
+        <!-- GCP SDK -->
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-storage</artifactId>
+            <version>0.5.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.datastax.cassandra</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
+            <version>${cassandra.driver.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mindrot</groupId>
+            <artifactId>jbcrypt</artifactId>
+            <version>0.3m</version>
+        </dependency>
+
+        <dependency>
+            <groupId>args4j</groupId>
+            <artifactId>args4j</artifactId>
+            <version>2.0.31</version>
+        </dependency>
```

Comment:
Okay for now, but I might work on switching to picocli (as the operator uses). It's a new lib with more features.

Reply:
👍